Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -1,26 +1,47 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection.Metadata.Ecma335;
+
+namespace BootCamp.Chapter
 {
     public class Inventory
     {
-        private Item[] _items;
-        public Item[] GetItems()
-        {
-            return new Item[0];
-        }
+        private const int defaultInventorySize = 10;
+
+        public List<Item> Items { get; private set; }
+        public int Size { get; private set; }
+        public int RemainingSpace { get; set; }
 
         public Inventory()
         {
-            _items = new Item[0];
+            Size = defaultInventorySize;
+            RemainingSpace = defaultInventorySize;
+            Items = new List<Item>(defaultInventorySize);
         }
 
-        public Item[] GetItems(string name)
+        public Item[] GetItem(string name)
```

Comment:
This method needs to either be named `GetItems` and return `Item[]`, or `GetItem` and return `Item`.

Reply:
Agree!

I've refactored this method so it returns an Item now. It lead to easier to read code in other classes that call this method.

I think the confusion came from the fact that some of the tests that are written for the original GetItems(string name) method in the Player class, expected an array as a result. It doesn't make much sense to be honest as it was designed to return a single item.

Anyways, this should be corrected now. GetItem() can return null as well but since there are plenty of null validations the code is still functional. :)